### PR TITLE
process_replay: migrate panda/peripheral/manager states

### PR DESCRIPTION
**Pull request by fredyshox**

### CODE REVIEW

File: selfdrive/test/process_replay/migration.py [hunk 1]
```diff
@@ -2,16 +2,57 @@
 
 from cereal import messaging
 from openpilot.selfdrive.test.process_replay.vision_meta import meta_from_encode_index
+from openpilot.selfdrive.car.toyota.values import EPS_SCALE
+from panda.python import Panda
```
Comment (adeebshihadeh) [inline selfdrive/test/process_replay/migration.py:6]:
```suggestion
from panda import Panda
```

File: selfdrive/test/process_replay/migration.py [hunk 2]
```diff
@@ -2,16 +2,58 @@
...
+    msgs = migrate_pandaStates(msgs, old_logtime)
   if camera_states:
     msgs = migrate_cameraStates(msgs)
 
   return msgs
 
+def migrate_pandaStates(lr, old_logtime=False):
+  all_msgs = []
+  # TODO: safety param migration should be handled automatically
+  safety_param_migration = {
```
Comment (fredyshox) [inline selfdrive/test/process_replay/migration.py:22]: @sshane is this still needed?

Comment (sshane) [reply selfdrive/test/process_replay/migration.py:22]: Can't regen the routes here to test (missing `wideCameraState`), but I'm pretty sure. controlsd checks that the safety params match, which will be different with our newer flags or EPS scaling factors.

Comment (sshane) [reply selfdrive/test/process_replay/migration.py:22]: FYI https://github.com/commaai/openpilot/pull/28829 removed the camera replay function that allowed old segments to work

Comment (fredyshox) [reply selfdrive/test/process_replay/migration.py:22]: For the wide cam: https://github.com/commaai/openpilot/pull/30143

**Comment by sshane** [comment]
Trying to regen the Prius route, but it doesn't enable. In the new segment I'm seeing planner errors, calibration incomplete, radar fault.

**Comment by sshane** [comment]
Can you post which PR this reverts partially? Helpful to go back through the history if we ever need to.

**Comment by fredyshox** [comment]
This was broken by behavioural change in https://github.com/commaai/openpilot/pull/28829. Before that regen did not use process replay and was manually replaying messages.

File: selfdrive/test/process_replay/migration.py [hunk 3]
```diff
@@ -2,16 +2,58 @@
...
+    "TOYOTA PRIUS 2017": EPS_SCALE["TOYOTA PRIUS 2017"] | Panda.FLAG_TOYOTA_STOCK_LONGITUDINAL,
```
Comment (sshane) [inline selfdrive/test/process_replay/migration.py:23]: realed that if you want to regen any other route that's a Prius this won't work. I think the real solution is taking the online fingerprinted carParams and updating the param pandaStates.

**Comment by sshane** [comment]
Can we get regen in CI first? This won't work on its own and there's a potentially better solution. Then we can merge fixes.

File: selfdrive/test/process_replay/migration.py (hunk 3 above)
Comment (fredyshox) [reply selfdrive/test/process_replay/migration.py:23]: hmm, process replay already runs fingerprinting to get CP, I could probably use that

**Pull request merged**